SensorHub_FW/src/measurement_grabber.hpp

```diff
@@ -56,6 +57,12 @@ void MeasurementGrabberTask(void* PvParameter) {
   while (1) {
     if (Data->Sensor != NULL) {
       data = Data->Sensor->GetSensorData();
+      static uint16_t lastSampleNum = data.sample_num;
+      if (data.sensor_id == 0X02 && data.sample_num == lastSampleNum + 1) {
+        data.num_of_bytes = 4;
```

Comment:
This can be a Macro, maybe it differs for each diff pressure sensor. Hard-coding values limits code re-usability